File: Day_21/Q81.cpp
#include <bits/stdc++.h>
using namespace std;
//Calculating The Length Of The String
int string_length(string str){
    int length = 0;
    while(str[length++] != '\0'){}
    return length-1;
}
int main(){
    string str;
    cin >> str;
    //Printing The Result
    cout << string_length(str);
    return 0;
}

File: Day_21/Q81.java
import java.util.*;
class Q81{
    //Calculating The Length Of The String
    static int string_length(String str){
        int length = 0;
        for(char ch : str.toCharArray()){
            length++;
        }
        return length;
    }
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        String str = sc.nextLine();
        //Printing The Result
        //Printing The Result
        System.out.print(string_length(str));
        sc.close();
    }
}